tufrepo/cli.py

```diff
@@ -98,9 +111,44 @@ def verify(ctx: Context, root_hash: Optional[str] = None):
 @cli.command()
 @click.pass_context
 def snapshot(ctx: Context):
-    """"""
-    repo = GitRepository(ctx.obj.keyring)
-    repo.snapshot()
+    """Update snapshot and timestamp meta information"""
+    ctx.obj.repo.snapshot()
+
+
+@cli.command()
+@click.pass_context
+@click.argument("role")
+def init_succinct_roles(ctx: Context, role: str):
+    """Create a new set of delegated bin files based on the succinct hash bin
+    delegation information in ROLE.
+
+    Note: it's required to call 'tufrepo edit ROLE add_succinct_delegation'
+    command first."""
+
+    targets: Targets
+    with ctx.obj.repo.edit(role) as targets:
+        if targets.delegations is None or targets.delegations.succinct_roles is None:
+            raise ValueError(
+                "Call 'tufrepo edit ROLE add_succinct_delegation' command first"
+            )
+
+        full_keys_info: List[PrivateKey] = []
+        for _ in range(targets.delegations.succinct_roles.threshold):
+            new_key: PrivateKey = ctx.obj.keyring.generate_key()
+            full_keys_info.append(new_key)
+            helpers.add_key(targets, role, None, new_key.public)
```

Comment:
this is a bit magical -- uses threshold info from the delegator but does not use any possibly existing keys from delegator, instead creates new ones.

I don't have a great alternative ideas yet so I think this is fine for now. I think a better solution would first require some smarter solution for the private key storage for succinct keys